src/agents/graph.py

```diff
@@ -483,7 +484,11 @@ async def astream(
                 self.handler,
                 UsageTrackerCallback(cluster_id, cast(IUsageMemory, self.memory)),
             ],
-            tags=[cluster_id],  # cluster_id as a tag for traceability and rate limiting
+            tags=[cluster_id],
+            metadata=get_langfuse_metadata(
+                message.user_identifier if message.user_identifier else "unknown",
```

Comment:
nit and more of an preference thingy
can be simplified to
```suggestion
                message.user_identifier or "unknown",
```

Reply:
updated